js/gallery/shared/Gallery.svelte

```diff
@@ -388,16 +388,14 @@
 			class:fixed-height={mode !== "minimal" && (!height || height == "auto")}
 			class:hidden={is_full_screen}
 		>
+			{#if interactive && selected_index === null}
```

Comment:
Moved this up here so that it's aligned with the top-right corner like we do in the other components

<img width="718" alt="image" src="https://github.com/user-attachments/assets/538a2948-c279-42e9-999c-f2006b33aa53">